src/pages/index.js

```diff
@@ -38,29 +49,38 @@ const profileDescriptionInput = document.querySelector(
 //Add modal
 const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#profile-add-modal");
+const EditAvatarModal = document.querySelector("#profile-avatar-modal");
+
+//Delete Modal
+//const cardDeleteModal = document.querySelector("#modal-delete-card");
+//const cardDeleteButton = document.querySelector(".card__delete-button");
 
 /*FormValidator.js*/
 
 const editFormElement = profileEditModal.querySelector(".modal__form");
 const addFormElement = addCardModal.querySelector(".modal__form");
+const avatarFormElement = EditAvatarModal.querySelector(".modal__form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms